src/utils/formatting-doc-to-txt.ts

```diff
@@ -5,93 +5,96 @@ const MAX_ANSWERS = 8;
 const inputFilename = 'input.txt';
 const outputFilename = 'output.csv';
 
-// Read the input file and split it into lines
-const lines = fs.readFileSync(inputFilename, 'utf8').split('\n');
-
-// Create an array to hold the data for the output CSV file
-var outputData = [];
-
-// Initialize the line index
-var i = 0;
-
-while (i < lines.length) {
-    // Check if this is a question line
-    if (lines[i].includes('.')) {
-       
-
-        var question = lines[i].trim().replace(',', '');
-
-        console.log(`Question detected: ${question}`);
-
-        var answers = [];
-        var points = [];;
-        // Loop through the next 5 lines to get the answers and points
-        for (var j = i + 1; j < i + MAX_ANSWERS; j++) {
-            try {
-                var line = lines[j].trim().replace('—', '').replace('â€”', '').replace('""', '').replace(',', '').replace("   ", '');
-                //var [answer, point] = line.split(' ');
-                const parts = line.split(' ');
-                if (parts.length >= 2) {
-                    var answer = parts.slice(0, -1).join(' '); 
-                    var point = parts[parts.length - 1];
-                }
-                else {
-                    var [answer, point] = line.split(' ');
-                }
-                console.log(`Answer: ${answer} Point: ${point}`);
-                if(answer == '' && point == undefined) {
+function writeOutputFile(filename, data) {
+    const header = ['question'];
+
+    for (let i = 0; i < MAX_ANSWERS; i++) {
+        header.push('answer');
+        header.push('points');
+    }
+
+    const csvRows = [header, ...data];
+    const csvContent = csvRows.map(row => row.join(',')).join('\n');
+
+    fs.writeFileSync(filename, csvContent, 'utf8');
+}
+
+function readInputFile(file) {
+    return fs.readFileSync(file, 'utf8').split('\n');
+}
+
+function processLines(lines) {
+    const outputData = [];
+    let i = 0;
```

Comment:
Rename i to somethig more significant to make the code more readable, something like fileParserIndex